docs/repositories-configure/ignoring-files.md

```diff
@@ -12,12 +12,17 @@ To exclude files from your repository analysis open your repository **Settings**
 
 ![Ignored files](images/ignored-files.png)
 
-You can also ignore files using your own [tool configuration files](configuring-code-patterns.md#using-your-own-tool-configuration-files), although this depends on the option being supported by each tool.
+!!! note
+    -   You can also ignore files using your own [tool configuration files](configuring-code-patterns.md#using-your-own-tool-configuration-files), although this depends on the option being supported by each tool.
+
+    -   {% include-markdown "../assets/includes/coverage-ignore.md" %}
 
 If you need more flexibility in ignoring files, use a Codacy configuration file to [define a custom list of file paths to exclude](codacy-configuration-file.md#syntax-for-ignoring-files).
 
-!!! note
-    {% include-markdown "../assets/includes/coverage-ignore.md" %}
+!!! important
+    If your repository has a [Codacy configuration file](codacy-configuration-file.md), the **Ignored files** settings defined on the Codacy UI don't apply. In this case, you must [ignore files using the configuration file](codacy-configuration-file.md#ignore-files) instead.
+
+    ![Ignored files](images/ignored-files-configuration-file.png)
```

Comment:
The screenshot in an admonition looks a bit weird. Do we even need it here? I don't think it adds a lot of value.

Reply:
I agree it's weird having the screenshot inside the admonition...
The screenshot gives the user a preview of what the screen looks like when the repository has a configuration file, which I believe is useful, and it was also the feedback I received.

Let me try taking it out of the admonition.

Reply:
Updated! Let me know if it makes sense.

I struggled a bit to find the right place for this piece of content... I initially thought of having it on the configuration file page, but I ended up deciding for this page, as it's where the users would come if they see this empty state on the screen instead of the settings. This way, they would know what they need to do in that case. 🤓

Reply:
As discussed, looking good!